tests/SideBySide/DebugOnlyTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using Xunit;
+
+namespace SideBySide
+{
+	public class DebugOnlyTests : IClassFixture<DatabaseFixture>
```

Comment:
The `SideBySide.DebugOnlyTests` class is only compiled when running with "Debug" configuration